hrui/hrui.php

```diff
@@ -542,24 +543,7 @@ function hrui_civicrm_alterContent( &$content, $context, $tplName, &$object ) {
 
   // fetch data to the new summary page UI
   if($context == 'page' && $tplName == "CRM/Contact/Page/View/Summary.tpl" ) {
-    $contact_id = CRM_Utils_Request::retrieve( 'cid', 'Positive');
-    /* $currentContractDetails contain current contact data including
-     * Current ( Position = $currentContractDetails->position ) and
-     * ( Normal Place of work =  $currentContractDetails->location )
-    */
-    $currentContractDetails = CRM_Hrjobcontract_BAO_HRJobContract::getCurrentContract($contact_id);
-    // $departmentsList contain current roles departments list separated by comma
-    $departmentsList = null;
-    if ($currentContractDetails)  {
-      $departmentsArray = CRM_Hrjobroles_BAO_HrJobRoles::getDepartmentsList($currentContractDetails->contract_id);
-      $departmentsList = implode(', ', $departmentsArray);
-    }
-    // $managersList contain current line managers list separated by comma
-    $managersList = null;
-    if ($currentContractDetails)  {
-      $managersArray = CRM_HRUI_Helper::getLineManagersList($contact_id);
-      $managersList = implode(', ', $managersArray);
-    }
+    $content = _hrui_updateContactSummaryUI($content);
```

Comment:
one last minor note ... since this is going to be appended to the end of $content variable , there is no need to pass $content to _hrui_updateContactSummaryUI function , so you can change this line to :

`$content .= _hrui_updateContactSummaryUI($content);`